build.sbt

```diff
@@ -399,6 +399,60 @@ lazy val flink = (project in file("flink"))
                                        "flink")
   )
 
+lazy val service = (project in file("service"))
```

Comment:
thanks for the separation so that our build will not be affected by the new deps.